app/src/main/java/com/babylon/wallet/android/domain/usecases/BuildDappResponseUseCase.kt

```diff
@@ -278,58 +279,62 @@ class BuildAuthorizedDappResponseUseCase @Inject constructor(
     }
 }
 
-class BuildUnauthorizedDappResponseUseCase @Inject constructor(
-    private val accessFactorSourcesProxy: AccessFactorSourcesProxy
-) : BuildDappResponseUseCase(accessFactorSourcesProxy = accessFactorSourcesProxy) {
+class BuildUnauthorizedDappResponseUseCase {
```

Comment:
previously we collected all the signatures at the end of the flow. Now as you can see the `accessFactorSourcesProxy` is gone because we ask the signatures on every screen (of the request item)